src/layouts/components/Editor/components/ImageBubbleMenu/ResizeImagePopover.tsx

```diff
@@ -65,7 +65,10 @@ export const ResizeImagePopover = ({
     editor
       .chain()
       .focus()
-      .setImage({
+      .updateAttributes("image", {
+        // use locally scoped style to override width in template scss
+        style: `width: ${data.value}%;`,
+        // To display the resized width value in the form
         width: `${data.value}%`,
```

Comment:
nit: I think this width should be removed since it is no longer needed, and we should adjust ResizeImagePopover to parse the `style` attribute.

Reply:
Thanks for the suggestion! Fixed in 88c13554923d35df10f6c39727505f547c759108 and be4b5a51a5447f92db613b49f92d5bbc5bd26621